test/integration/testPutGet.js

```diff
@@ -857,3 +861,315 @@ describe('PUT GET test multiple files', function () {
     setTimeout(checkResult, 100);
   }
 });
+
+describe('PUT GET test without compress', function () {
+  let connection;
+  const stage = `@${DATABASE_NAME}.${SCHEMA_NAME}.%${TEMP_TABLE_NAME}`;
+  const createTable = `create or replace table ${TEMP_TABLE_NAME} (${COL1} STRING, ${COL2} STRING, ${COL3} STRING)`;
+  const removeFile = `REMOVE ${stage}`;
+  const dropTable = `DROP TABLE IF EXISTS ${TEMP_TABLE_NAME}`;
+
+  // Create a temp file without specified file extension
+  const tmpFile = tmp.fileSync();
+  // Write row data to temp file
+  fs.writeFileSync(tmpFile.name, ROW_DATA);  
+  // Create a tmp folder for downloaded files
+  const tmpDir = fs.mkdtempSync(path.join(os.tmpdir(), 'get'));
+
+  before(async () => {
+    connection = testUtil.createConnection();
+    await testUtil.connectAsync(connection);
+    await testUtil.executeCmdAsync(connection, createTable);
+  });
+
+  after(async () => {
+    testUtil.deleteFileSyncIgnoringErrors(tmpFile);
+    testUtil.deleteFolderSyncIgnoringErrors(tmpDir);
+
+    await testUtil.executeCmdAsync(connection, removeFile);
+    await testUtil.executeCmdAsync(connection, dropTable);
+    await testUtil.destroyConnectionAsync(connection);
+  });
+
+  const tmpfilePath = getPlatformTmpPath(tmpFile.name);
+  const putQuery = `PUT ${tmpfilePath} ${stage} AUTO_COMPRESS=FALSE`;
+
+  const tmpdirPath = getPlatformTmpPath(tmpDir);
+  const getQuery = `GET ${stage} ${tmpdirPath}`;
+
+  const testCases =
```

Comment:
do we need testCases array when there are only one test?

Reply:
updated.